lib/puppet/provider/scheduled_task/win32_taskscheduler.rb

```diff
@@ -333,6 +343,17 @@ def translate_hash_to_trigger(puppet_trigger)
       self.fail "Unknown schedule type: #{puppet_trigger["schedule"].inspect}"
     end
 
+    if puppet_trigger['minutes_interval']
+      integer_interval = Integer(puppet_trigger['minutes_interval'])
+      self.fail 'minutes_interval must be an integer greater or equal to 0' unless integer_interval >= 0
```

Comment:
I would like to leave it as it is, because it matches the coding style of the rest of the checking in this file like above and below with things like self.fail "Must specify 'start_date' when defining a one-time trigger" unless puppet_trigger['start_date'] and self.fail "start_date must be on or after 1753-01-01" unless start_date >= Date.new(1753, 1, 1). Would that be ok?